great_expectations/profiler/domain_builder/simple_semantic_type_domain_builder.py

```diff
@@ -48,23 +53,26 @@ def _get_domains(
         elif isinstance(type_filters, Iterable):
             type_filters = [self.SemanticDomainTypes[x] for x in type_filters]
         else:
+            # TODO: <Alex>ALEX -- We should make this error message more informative.</Alex>
             raise ValueError("unrecognized")
         columns = validator.get_metric(MetricConfiguration("table.columns", dict()))
         domains = []
         for column in columns:
-            column_type = self._get_column_semantic_type(validator, column)
+            column_type = self._get_column_semantic_type(
+                validator=validator, column=column
+            )
             if column_type in type_filters:
                 domains.append(
                     {
                         # TODO: AJB 20210419 why is column just the column name string - will this be different based on execution engine versions?
                         # "domain_kwargs": {"column": column.name},
-                        "domain_kwargs": {"column": column},
-                        "domain_type": str(column_type),
+                        "domain_kwargs": {"column": column.name},
```

Comment:
When I was messing with the code earlier the actual name of the column was passed as the `column` variable so it was throwing an error trying to access the name via `column.name`. Should that be fixed somewhere else?

Reply:
@anthonyburdi This is not my change; my change is
```
                domains.append(
                    {
                        # TODO: AJB 20210419 why is column just the column name string - will this be different based on execution engine versions?
                        # "domain_kwargs": {"column": column.name},
                        "domain_kwargs": {"column": column.name},
                        "domain_type": column_type,
                    }
                )
```